autorest/date/unixtime_test.go

```diff
@@ -1,3 +1,5 @@
+// +build go1.7
```

Comment:
Sorry I wasn't clear on this, specifically the "t.Run" used in these tests was added in Go 1.7.  I don't think it's really worth re-writing these tests to fix that as we have coverage on later versions unless there's an easy way to do it.